qemu/tests/cfg/nvdimm.cfg

```diff
@@ -92,3 +93,13 @@
         - nvdimm_hotplug:
             del mem_devs
             target_mems = mem1
+        - nvdimm_nvml:
+            depends_pkgs = ndctl-devel daxctl-devel pandoc
+            nvml_test = yes
+            nvml_dir = /tmp/nvml
+            get_nvml = "git clone --depth=1 https://github.com/pmem/nvml.git ${nvml_dir}"
+            config_nvml = "cd ${nvml_dir}; make -j4"
```

Comment:
In the test case, there's a replacement branch if compiling failed, should we also include it ?

Reply:
I don't think so. If compile error, tester should intervene. Checking out to an older branch might miss some newly added test cases, so it should be the last option.